src/ReactiveDomain.Core/Json.cs

```diff
@@ -10,12 +10,29 @@
 using ReactiveDomain.Util;
 using Formatting = Newtonsoft.Json.Formatting;
 
-namespace ReactiveDomain.Messaging
+namespace ReactiveDomain
 {
     public static class Json
     {
-        private static IContractResolver GetContractresolver() {
-            var resolver = new DefaultContractResolver();
+        public class MetadataIgnoringResolver : DefaultContractResolver
+        {
+            protected override JsonProperty CreateProperty(MemberInfo member, MemberSerialization memberSerialization)
+            {
+                var property = base.CreateProperty(member, memberSerialization);
+
+                if (property.PropertyName == "_metadata")
+                {
+                    property.ShouldSerialize = i => false;
+                    property.Ignored = true;
+                }
+
+                return property;
+            }
+        }
+        private static IContractResolver GetContractresolver()
+        {
+            //Metadata is deserialized lazily with dedicated deserializers 
+            var resolver = new MetadataIgnoringResolver();
             resolver.DefaultMembersSearchFlags |= BindingFlags.NonPublic;
```

Comment:
This line gives a warning that "DefaultMembersSearchFlags is obsolete. To modify the members serialized inherit from DefaultContractResolver and override the GetSerializableMembers method instead."

Reply:
We can't fix that without re-doing all of serialization. That's out of scope right now.

Reply:
bummer, but OK